src/app/Main/Body/DataProtectionPage/DataProtectionPage.component.tsx

```diff
@@ -88,7 +88,6 @@ export function DataProtectionPageComponent(): JSX.Element {
                         <tr>{t("Chapter.1.AdressDP.Street")}</tr>
                         <tr>{t("Chapter.1.AdressDP.City")}</tr>
                         <tr>{t("Chapter.1.AdressDP.Phone")}</tr>
-                        <tr>{t("Chapter.1.AdressDP.Fax")}</tr>
                         <tr>
                             {t("Chapter.1.AdressBfR.Mail.Text")}{" "}
                             <a href="mailto:dsb@bfr.bund.de">
```

Comment:
I think you have to change the link too.
Also it would be nice if this comes from a config file or so.